src/Apps/Legion/Pages/StoreListPage/StoreListPage.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { IStoreListPageProps } from './StoreListPage.types';
+import { getDebugLogger } from '../../../../Models/Services/Utils';
+import StoreList from '../../Components/StoreList/StoreList';
+import { Stack } from '@fluentui/react';
+
+const debugLogging = false;
+const logDebugConsole = getDebugLogger('StoreListPage', debugLogging);
+
+const StoreListPage: React.FC<IStoreListPageProps> = () => {
+    // contexts
+
+    // state
+
+    // hooks
+
+    // callbacks
+
+    // side effects
+
+    // styles
+    logDebugConsole('debug', 'Render');
+
+    return (
+        <Stack tokens={{ childrenGap: 8 }}>
+            <h2>Welcome to the future</h2>
+            <StoreList />
+        </Stack>
+    );
+};
+
+export default StoreListPage;
```

Comment:
I don't think StoreList should be a "page", but should be a "component"

Reply:
It's really just so that we can have a consistent pattern for navigation. I suspect we will eventually have the page titles and possibly command bars or other information on the page that we wouldn't necessarily include in the list itself.